coffeecard/CoffeeCard.Models/DataTransferObjects/v2/Products/MenuItemResponse.cs

```diff
@@ -0,0 +1,32 @@
+using System.ComponentModel.DataAnnotations;
+
+namespace CoffeeCard.Models.DataTransferObjects.v2.Products
+{
+    /// <summary>
+    /// Represents a menu item that can be redeemed with a ticket
+    /// </summary>
+    /// <example>
+    /// {
+    ///    "id": 1,
+    ///    "name": "Cappuccino",
+    /// }
+    /// </example>
+    public class MenuItemResponse
+    {
+        /// <summary>
+        /// Id of menu item
+        /// </summary>
+        /// <value>Menu item Id</value>
+        /// <example>1</example>
+        [Required]
+        public int Id { get; set; }
+
+        /// <summary>
+        /// Name of menu item
+        /// </summary>
+        /// <value>Menu item name</value>
+        /// <example>Cappuccino</example>
+        [Required]
+        public string Name { get; set; } = string.Empty;
```

Comment:
We should not have a default value despite the compiler warning

Reply:
Addressed in 3dee1ab727bad46b91e42622e3b19b97b7b92bf8